source/aks/patching-artifactory.html.md.erb

```diff
@@ -1,152 +1,94 @@
 ---
 title: Artifactory Patching
 weight: 30
-last_reviewed_on: 2024-03-13
+last_reviewed_on: 2024-06-06
 review_in: 12 months
 ---
 
 # <%= current_page.data.title %>
 
-Artifactory is utilized in the context of managing and updating software dependencies and configurations within a Kubernetes environment. Here's how it is used based on the provided instructions:
+Artifactory is utilized in the context of managing and updating software dependencies and configurations within a Kubernetes environment.
 
-The steps below will show you how to check the current version of Artifactory being used (7.71.5) and then to update it to a newer version (7.77.3). This process involves accessing Artifactory's website to determine the latest version available.
+The steps below will show you how to check and update **self-hosted version** of Artifactory being used. This process involves accessing [Artifactory's website](https://jfrog.com/help/r/jfrog-release-information/artifactory-release-notes) to determine the latest **self-hosted version** version available.
 
-Before applying the update, a testing process is initiated. This involves downloading a repository containing configuration files for Artifactory (artifactory-sbox.yaml). This file is created and modified to reflect the desired changes, such as updating the version number.
+Before applying the update on PTL AKS Cluster, a testing process is initiated and is conducted on PTLSBOX AKS Cluster. This involves raising a PR to [GitHub](https://github.com/hmcts/cnp-flux-config) containing configuration files for Artifactory, creating 'artifactory-sbox.yaml' file and editing 'kustomization.yaml' files with your changes as seen in the example PR below.
```

Comment:
Was there anything else you did to test? Maybe just a one-liner to say you also check the pods are healthy with the newest image and whichever pipeline / app uses artifactory in ptlsbox is still working?

Reply:
just checking that the pods are healthy and able to access Artifactory after the update. Can add in a line 
`... ensuring Artifactory pods are healthy and access to Artifcatory (ptlsbox website) is accessible.`

Reply:
Something like this

`... editing 'kustomization.yaml' files with your changes as seen in the example PR below. In addition, checking Artifactory pods are healthy and access to [Artifcatory](https://artifactory.sandbox.platform.hmcts.net/ui/repos/tree/General/hmcts) is available.`

Reply:
Was there any checking the service itself?

Reply:
The only checking of the service was accessing Artifcatory site ensuring it was accessible.